### PR TITLE
Japanese translation of some pages

### PR DESCRIPTION
**Pull request by msakuta**
### Description

I translated 3 pages to Japanese and put links in the sidebar, following the style of #191.

* concepts/staking.md
* concepts/delegated-staking.md
* guides/explore-aptos.md

I put the translated pages as subpages of the original English as the same as #191, but I'm not sure if it's easy to navigate.
They look like below in the sidebar. It would look messy if you add more languages.

image

Since not many people seem to translate pages, I made a PR and waiting for comments.

### Checklist

- Do all Lints pass?
  - [] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/docusaurus/docs/concepts/delegated-staking/in-japanese.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+title: "日本語 (In Japanese)"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
```
Comment (gregnazario) [inline apps/docusaurus/docs/concepts/delegated-staking/in-japanese.md:6]: These are pretty in depth, have we migrated the associated docs @hariria

**Comment by gregnazario** [comment]
Hi, we plan on migrating all the docs to a new webpage, which will have built in ability to have internationalization support.

I suggest we hold off on converting any of them for the next month or so, and we'll then we can start translating them accordingly

**Comment by msakuta** [comment]
Thanks for the clarification!

To put some context, I have translated few pages incentivized by a hackathon reward, and since it had a short deadline, I could go only so far.
But now I get better idea of the whole documentation structure, I feel obligated to translate other relevant pages, because translating just few pages of the whole documentation doesn't make much sense.

If you have plan to integrate translation in a more organized way, I will be happy to wait for it.

**Comment by gregnazario** [comment]
> Thanks for the clarification!
> 
> To put some context, I have translated few pages incentivized by a hackathon reward, and since it had a short deadline, I could go only so far. But now I get better idea of the whole documentation structure, I feel obligated to translate other relevant pages, because translating just few pages of the whole documentation doesn't make much sense. Namely, terminology and internal links should work as a whole instead of individual pages.
> 
> If you have plan to integrate translation in a more organized way, I will be happy to wait for it.

Hello!

On the new docs site, we're supporting locale paths.

So, in this case https://preview.aptos.dev/jp would be the landing page for Japanese, when it's added.

A locale for japanese can be added as 'jp' here https://github.com/aptos-labs/developer-docs/blob/main/apps/nextra/docs.config.js#L17-L163

And then, all pages would just need to be translated accordingly in the `/jp` section https://github.com/aptos-labs/developer-docs/tree/main/apps/nextra/pages

You can feel free to reach out to me on telegram if you're still looking to do some translation, I'm sure we'd love the help!

**Pull request closed without merging**